playkit/src/main/java/com/kaltura/playkit/PlayerEvent.java

```diff
@@ -253,13 +257,63 @@ public String toString() {
         }
     }
 
+    public static class ConnectionAcquired extends PlayerEvent {
+
+        public final long connectDurationMs;
```

Comment:
Let's try to keep 3 values here, if it's not too complicated: DNS duration, connect duration, TLS duration.

Reply:
combine all to one object and updated the whole chain